docs/advanced-entity-analytics/turn-on-risk-engine.asciidoc

```diff
@@ -0,0 +1,80 @@
+// <<ers-requirements, Entity Risk Scoring prerequisites>>.
+
+The latest risk scoring engine, available in {stack} version 8.11 and newer, runs hourly to aggregate `Open` and `Acknowledged` <<alerts-ui-manage, alerts>> from the last 30 days, and assigns risk score to the host or user. It then aggregates the individual risk scores and normalizes them to a 0-100 range. The engine assigns a risk level by mapping the normalized risk score to one of these levels:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |Risk score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+|==============================================
+
+[discrete]
+=== Preview risky entities
+
+In {stack} version 8.11 and newer, you can preview risky entities before installing the latest risk engine. The preview shows the riskiest hosts and users found in the 1000 sampled entities during the time frame selected in the date picker.
+
+NOTE: The preview is limited to two risk scores per {kib} instance.
+
+To preview risky entities, go to **Manage** -> **Entity Risk Score**:
+
+[role="screenshot"]
+image::images/preview-risky-entities.png[Preview of risky entities]
+
+[discrete]
+=== Turn on the latest risk engine
+
+If you're running {stack} version 8.11 or newer, and you're installing the risk scoring engine for the first time:
+
+. Go to **Manage** -> **Entity Risk Score**.
+. Turn the **Entity risk scoring** toggle on.
+
+[role="screenshot"]
+image::images/turn-on-risk-engine.png[Turn on entity risk scoring]
+
+[NOTE]
+======
+* To view risk score data, you must have alerts generated in your environment.
+* If you previously installed the original <<user-risk-score, user>> and <<host-risk-score, host risk score>> modules, and you're upgrading to {stack} version 8.11 or newer, refer to <<upgrade-risk-engine, Upgrade to the latest risk engine>>.
+======
+
+[discrete]
+[[upgrade-risk-engine]]
+=== Upgrade to the latest risk engine
+
+If you upgraded to 8.11 from an earlier {stack} version, and you have the original risk engine installed, you can upgrade to the latest risk engine. You will be prompted to upgrade in places where risk score data exists, such as:
+
+* The Entity Analytics dashboard
+* The **User risk** tab on the Users page
+* The **User risk** tab on a user's details page
+* The **Host risk** tab on the Hosts page
+* The **Host risk** tab on a host's details page
+
+[role="screenshot"]
+image::images/risk-engine-upgrade-prompt.png[Prompt to upgrade to the latest risk engine]
+
+. Click **Manage** in the upgrade prompt, or go to **Manage** -> **Entity Risk Score**.
+. On the Entity Risk Score page, click **Start update** next to the **Update available** label.
++
+[role="screenshot"]
+image::images/risk-score-start-update.png[Start the risk engine upgrade]
+. On the confirmation message, click **Yes, update now!**. The old transform is removed and the latest risk engine is installed.
```

Comment:
The double punctuation is awkward; we probably don't need to reproduce the button's exuberant exclamation:
```suggestion
. On the confirmation message, click **Yes, update now**. The old transform is removed and the latest risk engine is installed.
```